commands/applications.go

```diff
@@ -21,9 +21,9 @@ func NewApplications(network *net.Network) (cmd *Applications) {
 
 func (cmd *Applications) Metadata() command_metadata.CommandMetadata {
 	return command_metadata.CommandMetadata{
-		Name:        "applications",
+		Name:        "application",
 		Description: "Show the status and location of running applications",
-		Usage:       "BROOKLYN_NAME applications",
```

Comment:
Ok, I see what you are doing now.  Is it worth some kind of comment to say that this is a subcommand?